cmd/cape/cmd/deploy.go

```diff
@@ -75,7 +76,7 @@ func init() {
 	rootCmd.AddCommand(deployCmd)
 
 	deployCmd.PersistentFlags().StringP("name", "n", "", "a name to give this function (default is the directory name)")
-	deployCmd.PersistentFlags().StringSliceP("pcr", "p", []string{""}, "pass multiple PCRs to validate against")
```

Comment:
why are you removing the PCR validation?

Reply:
@bendecoste Its ununsed, is this coming back? It was intelliJ cleaning up